python/cudf/cudf/__init__.py

```diff
@@ -69,6 +69,11 @@
 )
 from cudf.utils.dtypes import _NA_REP
 from cudf.utils.utils import set_allocator
+from cudf.api.extensions import (
+    register_dataframe_accessor,
+    register_series_accessor,
+    register_index_accessor,
+)
```

Comment:
I think you're missing a `cudf/api/__init__.py` file that imports `extensions` in order to make this work.